integrationTests/ts-esm/package.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
AFAIK, ESM changes only imports, it doesn't change any other aspect.
Please don't duplicate the entire test folder for it.
Just create a separate ESM-specific test file and alias ESM packages as `graphql-esm`.

Reply:
Please do the same for webpack